### ISSUE TITLE
Incompatability: Crashes when hovering over an enchanted trident

### ISSUE THREAD

**Issue by Ike-l** [Asek3/Rubidium#570]
When hover mouse over **enchanted** trident -> crash
This is an incompatability so not that important

[crash-client.txt](https://github.com/Asek3/Rubidium/files/12328426/crash-client.txt)

**Comment by WilliamBlaze** [comment]
yup! seems to be a common incompatibility as I had the same event triggered with an entirely different item. Anything with an Enchantment to be exact'

https://pastebin.com/jjnXgkE7
